packages/preset-uno/src/rules/behaviors.ts

```diff
@@ -60,3 +60,26 @@ export const appearance: Rule[] = [
     'appearance-none': 'none',
   }],
 ]
+
+// There is no match in the rule，like `bg`
+const unMatchedAbbreviation: Record<string, string> = {
+  bg: 'background-color',
+}
```

Comment:
I think we better maintain a map for known keys instead of bypassing everything

Reply:
I added some abbreviation attributes by referring to the existing rules. What do you think of this time~